tests/components/diagnostics/__init__.py

```diff
@@ -18,6 +18,8 @@ async def get_diagnostics_for_config_entry(hass, hass_client, config_entry):
 
 async def get_diagnostics_for_device(hass, hass_client, config_entry, device):
     """Return the diagnostics for the specified device."""
+    assert await async_setup_component(hass, "diagnostics", {})
+
```

Comment:
This is needed to make the tests pass. It was probably forgotten in the original PR...
cc @raman325

Reply:
This has already been addressed in https://github.com/home-assistant/core/pull/64504/commits/11dc2bcb3fc31ef8f027054f5ea954865171701e.

Reply:
Yes but that isn't merged yet - I can rebase if it gets merged before this one - but I don't think it will cause a conflict.